cmd/discovery/cmd.go

```diff
@@ -0,0 +1,48 @@
+// This package is a binary for trying out onvif discovery
+package main
+
+import (
+	"flag"
+
+	"github.com/viam-modules/viamrtsp/viamonvif"
+	"go.viam.com/rdk/logging"
+)
+
+func main() {
+	err := realMain()
+	if err != nil {
+		panic(err)
+	}
+}
+
+func realMain() error {
+	logger := logging.NewLogger("discovery")
+
+	debug := false
+	username := ""
+	password := ""
+
+	flag.BoolVar(&debug, "debug", debug, "debug")
+	flag.StringVar(&username, "user", username, "username")
+	flag.StringVar(&password, "pass", password, "password")
+
+	flag.Parse()
+
+	if debug {
+		logger.SetLevel(logging.DEBUG)
+	}
+
+	list, err := viamonvif.DiscoverCameras(username, password, logger, flag.Args())
```

Comment:
Should we document that the additional args represents a list of net interface names?

Reply:
```suggestion
	// Additional non-flag args are net interface names to use for discovery
	// If none are provided, we will discover over all available net interfaces on the machine.
	list, err := viamonvif.DiscoverCameras(username, password, logger, flag.Args())
```

Reply:
Also, do you think it'd be useful to expose this new net interface param to users? Or was your thinking behind adding this mainly for debugging use cases?